src/modules/dashboard/components/CoinMachine/CoinMachine.css

```diff
@@ -60,12 +61,14 @@
 
 .sales {
   composes: container;
-  min-height: 100%;
+  margin-bottom: 50px;
+  height: calc(100% - 50px);
   grid-column: buy / comments;
   grid-row: sales / last-line;
 }
 
 .comments {
+  margin: 0 30px 50px 0;
```

Comment:
Same here - why do we apply on one of the elements only and not on the larger container?

Reply:
Done!